en/02_Developer_Guides/14_Files/05_File_Manipulation.md

```diff
@@ -0,0 +1,419 @@
+---
+title: File manipulation
+summary: Learn how to manipulate file records in code
+icon: file-medical-alt
+---
+
+# File manipulation
+
+Asset storage is provided out of the box via a [Flysystem](https://flysystem.thephpleague.com/docs/) backend store. This abstraction allows for files to be stored in any number of different ways, such as storing them in the cloud, so you cannot rely on having a local file path in order to get and manipulate the contents of any given asset.
+
+Silverstripe CMS provides a well-abstracted API for creating, manipulating, and storing assets.
+
+See [images](./images/) for some image-specific manipulation methods.
+
+## Creating new files in PHP
+
+When working with files in PHP you can upload a file into a [`File`](api:SilverStripe\Assets\File) dataobject
```

Comment:
Don't have a better suggestion on hand at the moment but 'upload a file into a File dataobject' doesn't sit well with me. It's not 'upload' per se and it's not 'into' the DO. Can we find a better wording?

Reply:
That was already the wording being used - I'm not going to change it in this PR.